### ISSUE TITLE
Trezor Plugin - Unable to activate

### ISSUE THREAD

**Issue by Thireus** [spesmilo/electrum#1371]
(Tested on Mac OS X 10.10.4)

The Trezor plugin cannot be activated via the Electrum plugin settings:

```
Jul 19 02:01:36 dev.local electrum[74166] <Notice>: Traceback (most recent call last):
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:   File "/Applications/Electrum.app/Contents/Resources/lib/python2.7/gui/qt/main_window.py", line 2803, in <lambda>
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:     return lambda: do_toggle(cb, name, w)
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:   File "/Applications/Electrum.app/Contents/Resources/lib/python2.7/gui/qt/main_window.py", line 2794, in do_toggle
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:     p.enable()
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:   File "/Applications/Electrum.app/Contents/Resources/lib/python2.7/lib/plugins.py", line 164, in enable
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:     self.set_enabled(True)
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:   File "/Applications/Electrum.app/Contents/Resources/lib/python2.7/plugins/trezor.py", line 77, in set_enabled
Jul 19 02:01:36 dev.local electrum[74166] <Notice>:     self.wallet.storage.put('use_' + self.name, enabled)
Jul 19 02:01:36 dev.local electrum[74166] <Notice>: AttributeError: 'NoneType' object has no attribute 'storage'
```

**Comment by Thireus** [comment]
This issue happens when restoring a Trezor wallet from its seed. The plugin should not be disaplyed in the plugins menu (but instead should remain gray for this kind of wallet).

What's happening here is that Electrum things the wallet is directly attached to the Trezor device (which it is not), and thus allows users to enable the Trezor plugin to be plugins settings (which it should not).

To fix this issue: A wallet of Trezor type but restored from Trezor seed should leave the Trezor Plugin grayed in the plugin settings and not let users enable it.

**Comment by ecdsa** [comment]
fixed in e18ddd0096250d6ebcda2bc16a9ea49335d0e7c7
